SU-HSDO/suhumsci
# updated blt to 10.x-dev

# READY FOR REVIEW

# Summary
- Updated BLT & Drush

# Needed By (Date)
- some day

# Urgency
- low

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. try out some blt and drush commands to ensure it still works.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
